data/map planets.txt

```diff
@@ -4692,7 +4692,7 @@ planet Violetwake
 planet "Vivid Aether"
 	attributes "gas giant" "requires: gaslining" uninhabited
 	landscape land/nasa29
-	description `Whether it be from local evolution or panspermia, Vivid Aether's clouds are home to colonies of anaerobic microbial life, feeding on both sunlight and traces of methane in the atmosphere. These form colorful streaks in the upper atmosphere, which were what initially attracted Saryd explorers to the planet. Now, it is common for Heliarch research patrols to sit in orbit for weeks at a time in order to study the simple, yet exotic, ecology. Occasionally, they send down hardened research vessels capable of withstanding the extreme pressures, though none happen to be nearby right now.`
+	description `Whether it be from local evolution or panspermia, Vivid Aether's clouds are home to colonies of anaerobic microbial life, feeding on both sunlight and traces of methane in the atmosphere. These form colorful streaks in the upper atmosphere, which were what initially attracted Saryd explorers to the planet. Now, it is common for Heliarch research patrols to sit in orbit for weeks at a time in order to study the simple, yet exotic, ecology. Occasionally, they send down hardened research probes capable of withstanding the extreme pressures, though none survive for very long.`
```

Comment:
Not being able to survive for very long would mean the probes are pretty frail, which doesn't work with the Heliarchs having extremely sturdy hulls; even if they make use of civilian-grade materials for the probes, the Arach ships have pretty good hulls in terms of HP, so they'd still get a good amount of time from the probes.
If the implication is that they willingly just drop down the probes and let them gather info until they fall down sufficiently that they start to break and malfunction, that'd be oddly wasteful of them, and if that were the case I'd rather that be explained, but we don't have much working room with the description box as is, so I'd leave the current end bit as is, because that'd make it so it can still be that they drop the probes with intent of just letting them get info as long as they can before breaking down, it's just the player doesn't see any as they're very reduced in number and only ever dropped occasionally, rather than putting the capabilities and usefulness of these probes and the process of sending them down into question.

Alternatively, could maybe cut the paragraph at "pressures".

```suggestion
	description `Whether it be from local evolution or panspermia, Vivid Aether's clouds are home to colonies of anaerobic microbial life, feeding on both sunlight and traces of methane in the atmosphere. These form colorful streaks in the upper atmosphere, which were what initially attracted Saryd explorers to the planet. Now, it is common for Heliarch research patrols to sit in orbit for weeks at a time in order to study the simple, yet exotic, ecology. Occasionally, they send down hardened research probes capable of withstanding the extreme pressures, though none happen to be nearby right now.`
```

Reply:
```suggestion
	description `Whether it be from local evolution or panspermia, Vivid Aether's clouds are home to colonies of anaerobic microbial life, feeding on both sunlight and traces of methane in the atmosphere. These form colorful streaks in the upper atmosphere, which were what initially attracted Saryd explorers to the planet. Now, it is common for Heliarch research patrols to sit in orbit for weeks at a time in order to study the simple, yet exotic, ecology. Occasionally, they send down hardened research probes capable of withstanding the extreme pressures, though none can stay for long.`
```

How about this? It removes the implications of their destruction, and retains the important detail about not being able to loiter indefinitely.

Reply:
`none happen to be nearby right now` seems odd when referring to probes. 
Is that saying that you're inside the gas giant, and there aren't any probes around? Or that there are none nearby to the gas giant at all? 

If it's saying that there aren't any near you, that doesn't seem relevant to point out, given its exceptionally unlikely one would happen to be nearby. 
If it's saying that there aren't any near the gas giant, that implies the probes operate independently. 

I'd suggest just cutting the sentence off at "pressures" or keeping the line as it is and changing "probes" back to "vessels"

Reply:
> During my discussion with Arachi, the intent was explicitly for the Heliarchs to be using small crewed vessels for up-close examinations of the atmospheric ecosystem.

In that case, how does `Occasionally, they send down hardened research vessels capable of withstanding the extreme pressures, though none happen to be nearby right now.` sound?
That makes it more explicitly "vessels", avoids the weirdness of implying that probes are operating independently and clarifies what it means by "none happen to be nearby", since a crewed vessel *would* be of note, unlike a probe.

Reply:
```suggestion
	description `Whether it be from local evolution or panspermia, Vivid Aether's clouds are home to colonies of anaerobic microbial life, feeding on both sunlight and traces of methane in the atmosphere. These form colorful streaks in the upper atmosphere, which were what initially attracted Saryd explorers to the planet. Now, it is common for Heliarch research patrols to sit in orbit for weeks at a time in order to study the simple, yet exotic, ecology.`
```

When you get right down to it, there'd have to be thousands, perhaps even millions, of ships of any kind floating around in here in order to have the player just randomly pop down close to one.

So how about this? We just remove that last sentence entirely. No mentions at all.

Reply:
Oh, yes. You are correct; looking at it again, that is indeed the original wording. 
So I think that is what we should stick with. The wording seems good as it is, I don't see any need to change it, and all of the suggested changes thus far seem to be worse than the original.

Reply:
All options here have been negated by simply deleting the sentence. After all, the odds of the player running into *any* ship down here, of any kind, are astronomically small to the point of impossibility. So better to avoid all the fuss and just skip establishing anything either way.